phenol-core/src/main/java/org/monarchinitiative/phenol/formats/mpo/MpAllelicComposition.java

```diff
@@ -0,0 +1,85 @@
+package org.monarchinitiative.phenol.formats.mpo;
+
+import java.util.Objects;
+
+/**
+ * This class represents the Allelic Composition field of the MGI_PhenoGenoMP.rpt file as a string (without breaking
```

Comment:
MGI_PhenoGenoMP.rpt is not the only file that has an Allelic Composition field, maybe reword this description to be more general?

Reply:
I think MGI_GenePheno.rpt also has this field. But please do not accept this PR for right now, I think we need a few more tests and the documentation needs to be improved a bit!